versions/ufo3/lib.plist.md

```diff
@@ -36,6 +36,24 @@ This data is optional.
 
 [The OpenType meta table specification.]
 
+#### public.openTypeCategories
+
+This key is used to define the category of the glyphs to be used, for example, as glyph class in the [OpenType GDEF Glyph Class Definition Table]. The categories are stored in a dictionary keyed by glyph name, each value must be one of `base`, `mark`, `ligature` or `component`. This data is optional.
+
```

Comment:
Should state the at the glyph name may not be in the font, and that all glyphs in the font do not need to appear in the list.

Reply:
would say then "must be only one of" and "glyph names can only appear once" (yes, the data structure does imply this, I know)